monitor/resource/config.yml

```diff
@@ -1,27 +1,59 @@
 temperatureF:
   id: 1
-  name: Temperature F
+  name:
+    en: Temperature F
+    de: Temperatur F
   min: 95
   max: 102
-  errorMessage: Temperature is critical!
+  error_message:
+    en: Temperature is out of range!
+    de: Temperatur ist außerhalb des Bereichs!
+  warning:
+    enabled: true
+    tolerance: 1.5
+    low_warning:
+      en: Approaching hypothermia!
```

Comment:
It's nice to have all the configuration in one place. Only, one of the downsides may be the contributors to this file. E.g., if we need to send all the strings to a translation agency and get it back in a new language - will the agency need to learn yml?

Reply:
Oh, one more thing: When multiple people handle a file this way, there are always chances of errors (think of merge conflicts, etc). Errors in this file are not caught by the compiler, they need to be caught at the time of parsing. Can you think of some 'static checks' to catch the common issues in this file?
They would be 'static' because they can be run without building or running the code, which makes them integrate-able in the IDE, for example.